yarn.lock

```diff
@@ -3486,6 +3486,7 @@ __metadata:
     "@nuxtjs/sentry": latest
     "@nuxtjs/style-resources": latest
     "@nuxtjs/toast": latest
+    "@rsksmart/rif-aggregation-sdk-js": "https://github.com/rsksmart/ri-aggregation"
```

Comment:
I think that this yarn.lock doesn't reflect the latest changes, so we should update it also

Reply:
Updated